framework/auth/views.py

```diff
@@ -26,10 +26,10 @@
 from website import mails
 from website import language
 from website import security
+from website.project.views.contributor import throttle_period_expired
```

Comment:
Minor: `throttle_period_expired` doesn't really belong in `views.contributor`. It'd be better to go in `website.utils.time`. Feel free to make this change. Or not. Up to you.